content/200-concepts/300-more/400-comparisons/04-prisma-and-drizzle.mdx

```diff
@@ -14,20 +14,229 @@ This page compares Prisma and Drizzle.
 
 While Prisma and Drizzle solve similar problems, they work in very different ways and have individual pros and cons. Which one to choose will depend on the need of your project and the exact tradeoffs that are important for it.
 
-**Drizzle** is a traditional SQL query builder which lets you compose SQL queries with JavaScript/TypeScript functions. It can be used to query a database or run migrations.
+**Drizzle** is a traditional SQL query builder which lets you compose SQL queries with JavaScript/TypeScript functions. It can be used to query a database or run migrations. Drizzle also offers a Queries API which offers a higher level abstraction from SQL and is useful for nested relations. Drizzle schema is defined in TypeScript files which are used to generate SQL migrations and are then executed against a database.
```

Comment:
```suggestion
**Drizzle** is a traditional SQL query builder that lets you compose SQL queries with JavaScript/TypeScript functions. It can be used to query a database or run migrations. Drizzle also offers a Queries API, which offers a higher level abstraction from SQL and can be used to read nested relations. Drizzle schema is defined in TypeScript files, which are used to generate SQL migrations and are then executed against a database.
```